satnogsclient/upsat/packet.py

```diff
@@ -136,7 +137,7 @@ def ecss_packetizer(ecss, buf):
     buf[1] = app_id_ls
     seq_flags = packet_settings.TC_TM_SEQ_SPACKET
     seq_count = ecss['seq_count']
-    seq_count_ms = seq_count & 0xFF00
+    seq_count_ms = (seq_count >> 8) & 0x00FF
     seq_count_ls = seq_count & 0x00FF
     seq_count_ms = seq_count >> 8
```

Comment:
This line (142) needs to be deleted since the right shift has already happened in line 140